jobs/openshift/cluster/create/openshift-cluster-create.yaml

```diff
@@ -8,9 +8,12 @@
           name: 'clusterName'
           default: ''
           description: '[REQUIRED] The name of the cluster to create'
-      - string:
+      - choice:
           name: 'awsRegion'
-          default: 'eu-west-2'
+          choices:
+            - 'us-east-1'
+            - 'eu-west-1'
+            - 'eu-west-2'
```

Comment:
I'm not actually sure selecting anything other than "eu-west-2" works at the moment does it?

Reply:
Just wanted to use the same values used in the destroy job

Reply:
What's the default value set to if no awsRegion is specified? Can you set the default choice to be "eu-west-2", i'm thinking of jenkins jobs that trigger this one, but might just use the default values.

Reply:
ok let me set it

Reply:
I guess this is the only job that i know of that triggers it, and it's specifying the region https://github.com/integr8ly/ci-cd/blob/master/jobs/openshift/cluster/integreatly/test/Jenkinsfile#L30